test/binary_ops.hpp

```diff
@@ -85,7 +85,7 @@ struct Binary<T, AF_BINARY_MAX>
     template<>                                      \
     struct Binary<T, AF_BINARY_MIN>                 \
     {                                               \
-        T init()                                    \
+        static T init()                                    \
```

Comment:
The backslash is not formatted correctly.

Reply:
Yup I'll fix these slashes

Reply:
👍